war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -0,0 +1,463 @@
+      List<DataDocument> users = document.getArrayList(LumeerConst.Organization.ATTR_USERS, DataDocument.class);
+      return users != null ? users.stream().collect(Collectors.toMap(d -> d.getString(LumeerConst.Organization.ATTR_USERS_USERNAME), d -> d.getArrayList(LumeerConst.Organization.ATTR_USERS_USER_ROLES, String.class))) : null;
+   }
+
+   /**
+    * Reads a list of organizations the user is assigned to.
+    *
+    * @param userName
+    *       name of the user
+    * @return list of organizations
+    */
+   public Map<String, String> readUserOrganizations(final String userName) {
+      List<DataDocument> documents = dataStorage.searchIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationFilterByUser(userName), Arrays.asList(LumeerConst.Organization.ATTR_ORG_ID, LumeerConst.Organization.ATTR_ORG_NAME));
+      return documents != null ? documents.stream().collect(Collectors.toMap(d -> d.getString(LumeerConst.Organization.ATTR_ORG_ID), d -> d.getString(LumeerConst.Organization.ATTR_ORG_NAME))) : null;
+   }
+
+   /**
+    * Adds user to organization with specific user roles.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param userName
+    *       name of the user to add
+    * @param userRoles
+    *       organization roles of the given user
+    */
+   public void addUserToOrganization(final String organizationId, final String userName, final List<String> userRoles) {
+      DataDocument document = new DataDocument(LumeerConst.Organization.ATTR_USERS_USERNAME, userName)
+            .append(LumeerConst.Organization.ATTR_USERS_USER_ROLES, userRoles);
+      dataStorage.addItemToArray(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), LumeerConst.Organization.ATTR_USERS, document);
+   }
+
+   /**
+    * Adds user to organization with default roles.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param userName
+    *       name of the user to add
+    */
+   public void addUserToOrganization(final String organizationId, final String userName) {
+      addUserToOrganization(organizationId, userName, readDefaultRoles(organizationId));
+   }
+
+   /**
+    * Removes the given user from the organization.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param userName
+    *       name of the user to remove from organization
+    */
+   public void removeUserFromOrganization(final String organizationId, final String userName) {
+      List<String> userRoles = readUserRoles(organizationId, userName);
+      DataDocument userDocument = new DataDocument(LumeerConst.Organization.ATTR_USERS_USERNAME, userName)
+            .append(LumeerConst.Organization.ATTR_USERS_USER_ROLES, userRoles);
+      dataStorage.removeItemsFromArray(LumeerConst.Organization.COLLECTION_NAME, userFilter(organizationId, userName), LumeerConst.Organization.ATTR_USERS, Collections.singletonList(userDocument));
+   }
```

Comment:
Unnessary reading of user roles. Look at `ProjectFacade.dropUserFromProject` method for way how to implement it correctly.